src/spec/Footer.spec.jsx

```diff
@@ -0,0 +1,28 @@
+import { render, screen } from "@testing-library/react";
+import { beforeEach, describe, expect } from "vitest";
+
+import OPTIMIZED_SITE_LIST from "../utils/optimizedSiteList";
+import Footer from "../components/Footer/Footer";
+
+describe("Footer", () => {
+  let getByText;
+
+  beforeEach(() => {
+    const rendered = render(<Footer />);
+    getByText = rendered.getByText;
+  });
+
+  it("Footer has shown all OptimizedSites", () => {
+    OPTIMIZED_SITE_LIST.forEach((OptimizedSite) => {
+      const linkElement = getByText(OptimizedSite.siteName);
+
```

Comment:
```suggestion
  beforeEach(() => {
    render(<Footer />);
  });

  it("Footer has shown all OptimizedSites", () => {
    OPTIMIZED_SITE_LIST.forEach((OptimizedSite) => {
      const linkElement = screen.getByText(OptimizedSite.siteName);
```

getByText를 아래 테스트코드에서 작성하신 것 처럼 screen.getByText로 바꾸시면
beforeEach에서 불필요한 변수 재할당을 없애고 통일성을 줄 수 있을 것 같습니다!

Reply:
수정 완료했습니다!